tasks/identity.yml

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
this doesn't look like it's in the right folder - it should be in the `roles/{conjur-host-identity}` directory, right? I'm guessing that'll apply to a bunch of files below, too